packages/miniflare/package.json

```diff
@@ -46,6 +46,7 @@
 		"acorn": "8.14.0",
 		"acorn-walk": "8.3.2",
 		"exit-hook": "2.2.1",
+		"get-port": "^7.1.0",
```

Comment:
This should be a `devDependency`, so that miniflare correctly bundles it

Reply:
sorry my bad, fixed 🙂👍